### PR TITLE
Add page for pandoc

### PR DESCRIPTION
**Pull request by edgester**

### CODE REVIEW

File: pandoc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Convert HTML file to Markdown
+pandoc -f html -t markdown  -i <HTML input file> -o  <markdown output file>
```
Comment (EmilySeville7cfg) [inline pandoc:2]:
```suggestion
---
syntax: bash
tags: [converters, markdown]
---

# Convert HTML file to Markdown:
pandoc --from=html --to=markdown  --incremental <path/to/html_input_file> --output=<path/to/markdown_output_file>
```

Mandatory fix: add YAML front-matter.
Optional fix: add trailing colon.
Optional fix: use long options as they are more memorable.
Optional fix: standardize placeholder format.

File: pandoc [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+# Convert Markdown file to HTML
+pandoc -f markdown -t html -i <markdown input file>  -o <HTML output file>
```
Comment (EmilySeville7cfg) [inline pandoc:5]:
```suggestion
# Convert Markdown file to HTML
pandoc --from=markdown --to=html --incremental <path/to/markdown_input_file> --output=<path/to/html_output_file>
```

Almost the same issue as above.

**Review by EmilyGraceSeville7cf — CHANGES_REQUESTED** [review]
Mandatory fixes found, changes requested.
Thanks for a contribution! 🚀

**Comment by chrisallenlane** [comment]
Thanks!

**Pull request merged**